ispyb/model/container.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import, division, print_function
+
+import ispyb
+import ispyb.model
+
+
+class Container(ispyb.model.DBCache):
+    """An object representing a Container database entry. The object
+    lazily accesses the underlying database when necessary and exposes record
+    data as python attributes.
+    """
+
+    def __init__(self, containerid, db_conn, preload=None):
+        """Create a Container object for a defined ContainerID. Requires
+        a database connection object exposing further data access methods.
+
+        :param containerid: ContainerID
+        :param db_conn: ISPyB database connection object
+        :return: A Container object representing the database entry for
+                 the specified ContainerID
+        """
+        self._db = db_conn
+        self._containerid = int(containerid)
+        if preload:
+            self._data = preload
+
+    def reload(self):
+        """Load/update information from the database."""
+        raise NotImplementedError("Loading containers is not currently supported")
+
+    @property
+    def containerid(self):
+        """Returns the ID associated with this container information."""
+        return self._containerid
+
+    def __bool__(self):
+        """Container object evaluates to True in a boolean context if grid
```

Comment:
Grid information? That sounds a bit strange for containers?

Reply:
only saw this now. fixed in 4bb6496b534f3de3742a8545196b6eb47a0fddbe